test/functional/p2p_feefilter.py

```diff
@@ -88,24 +88,22 @@ def test_feefilter(self):
 
         conn = self.nodes[0].add_p2p_connection(TestP2PConn())
 
-        # Test that invs are received by test connection for all txs at
-        # feerate of .2 sat/byte
+        self.log.info("Test that txs paying .2 sat/byte are being received by test connection")
         node1.settxfee(Decimal("0.00000200"))
         txids = [node1.sendtoaddress(node1.getnewaddress(), 1) for x in range(3)]
         assert allInvsMatch(txids, conn)
         conn.clear_invs()
 
-        # Set a filter of .15 sat/byte on test connection
+        self.log.info("Set a fee filter of .15 sat/byte on test connection")
```

Comment:
I think you can leave this line as a code comment instead of logging it.

For the logging in general, would you mind prefixing the decimal values with 0? e.g. s/.15/0.15/

This is personal taste, so feel free to ignore, but I think it's unnecessarily verbose to add "that" after "Test..."

Reply:
Done.